### PR TITLE
Aya week1 hm

### PR DESCRIPTION
**Pull request by Aya-Alabrash**

### CODE REVIEW

File: Week1/homework/world-queries.txt [hunk 1]
```diff
@@ -1,40 +1,250 @@
...
+| Brazil             |
+| Australia          |
+| India              |
+| Argentina          |
+| Kazakstan          |
++--------------------+
+10 rows in set (0.00 sec)
+
+OR..
+mysql> SELECT name FROM country GROUP BY SurfaceArea DESC LIMIT 10;
```
Comment (pcelentano) [inline Week1/homework/world-queries.txt]: 😱not even sure whow this works 😅  first query is way more clear and also easier for the db to make (and easier to optimize)1

File: Week1/homework/world-queries.txt [hunk 2]
```diff
@@ -1,40 +1,250 @@
...
+| Popluation of Asia |
++--------------------+
+|         3705025700 |
++--------------------+
+1 row in set (0.07 sec)
+--------------------------------------------------------------
+
+8.Which languages are spoken in the Region of �South America�
+
+mysql> SELECT distinct language FROM countrylanguage as cl JOIN country as c
```
Comment (pcelentano) [inline Week1/homework/world-queries.txt]: Nice job with the table aliases !

File: Week1/homework/world-queries.txt [hunk 3]
```diff
@@ -1,40 +1,250 @@
...
+| Sranantonga      |
+| Goajiro          |
+| Warrau           |
++------------------+
+21 rows in set (0.05 sec)
+--------------------------------------------------------------
+
+9.What are the languages spoken on all cities named �Barcelona� (you may need to join 3 tables =) )
+
+mysql> SELECT distinct language FROM countrylanguage as cl JOIN city as c
```
Comment (pcelentano) [inline Week1/homework/world-queries.txt]: This works as advertised 👏 ... but part of the fun was realising there was more than one city named Barcelona ! 
I was curious so i gave this one a try !
```sql
SELECT
  ci.name,
  co.name,
  l.language
FROM country co
  JOIN city ci on ci.countrycode = co.Code
  JOIN countrylanguage l ON l.CountryCode = co.Code
WHERE ci.Name = 'Barcelona';
```

**Comment by Aya-Alabrash** [comment]
@pcelentano thank you :)

**Pull request closed without merging**